.gitignore

```diff
@@ -64,21 +64,16 @@ typings/
 .csr
 .pem
 
-# CSS files are compiled
-public/styles.css
-public/styles.css*
+# Compiled CSS and files
+public/dist
 
 .DS_Store
 credentials
 cdk.out
 cdk.out
-
 cdk/local-image
-credentials
-
-
-
-public/js/dist
 
 # Visual Studio Code
-.vscode
\ No newline at end of file
+.vscode
+
+credentials
```

Comment:
Why is credentials multiple times?

Reply:
is this an upstream merge or something you added?

Reply:
That's upstream... looks like it's been there at least since the previous release